packages/react-components/form-elements/src/Switch/Input.tsx

```diff
@@ -0,0 +1,22 @@
+import { css } from '@emotion/react';
+import { InputHTMLAttributes } from 'react';
+
+// Checkbox is visually hidden but remains accessible to screen readers
+const inputStyle = css({
+  border: 0,
+  height: '100%',
+  left: 0,
+  margin: 0,
+  opacity: 0.001,
```

Comment:
It might be good to add a note that this is for a11y

Reply:
There is a comment at the top of styles :)

Reply:
Sorry, I wasn't very clear, I meant about this almost 0 number, since many places say that 0 should also work.

Reply:
I think it is safe to assume that it will work everywhere. There may be a risk `opacity: 0` removes an element from screen readers completely similar to how `display: none` does.